DatabaseConnector.Rproj

```diff
@@ -1,4 +1,5 @@
 Version: 1.0
+ProjectId: 9d51e576-41a3-432f-b696-8bfdc3eed676
```

Comment:
This ProjectId appears to be something introduced in the latest version of RStudio.